server/backend/api/app/routers/router_guards.py

```diff
@@ -414,3 +427,13 @@ async def enforce_bceid_by_same_org_guard(
                 error_code=ERROR_CODE_DIFFERENT_ORG_GRANT_PROHIBITED,
                 error_msg="Managing for different organization is not allowed.",
             )
+
+
+def enforce_bceid_terms_conditions_guard(
+    requester: Requester = Depends(get_current_requester),
+):
+    if requester.requires_accept_tc:
+        utils.raise_http_exception(
+            error_code=ERROR_CODE_TERMS_CONDITIONS_REQUIRED,
+            error_msg="Requires to accept terms and conditions.",
+        )
```

Comment:
The router_guard checks for T&C for use in backend api endpoints for BCeID delegated admin user.
Exception has default http error status: 400.